### ISSUE TITLE
[CHANGE] Spell out accessibility statements

### ISSUE THREAD

**Issue by mgifford** [gc-da11yn/gc-da11yn.github.io#208]
"2.3.2.3 All existing public-facing web pages must feature an accessibility statement and a user feedback mechanism;" - https://a11y.canada.ca/en/standards/standard/#2.3.2.3

This is a good step, but spell it out. 

Just take what the UK did & translate it to the Canadian context - https://www.gov.uk/government/publications/sample-accessibility-statement

Feedback also needs to be collected and aggregated. There should be an expectation that if a citizen reports an error that they:
- get a confirmation message immediately
- have an expectation that if they provide an email that they will hear back in a reasonable time period about the error that was reported.
- If the issue is replicated but cannot be fixed, it needs to be incorporated in the accessibility statement until it is. If a workaround is available, that should be described. 

Also bring in what The Netherlands is doing by scanning government sites to see that they comply. Then publish that data. 

You can't just say, "Have an accessibility statement". Departments have had those, with a contact us page for over a decade. It doesn't change things.

**Comment by mgifford** [comment]
"2.3.7.1 Departments must provide mechanisms for users to give feedback on the accessibility of ICT."

All the information above does apply to 2.3.7.1 too.

**Comment by mmiron-HC** [comment]
What I would like to know is where this statement should reside.

**Comment by mgifford** [comment]
Most implementations of accessibility statements that I have seen include a link on every page of the website. Just like privacy, or terms of service documents. We have no idea where citizens will land on your webpage, but where people expect to find information about accessibility is in the footer of the page. This is the norm. 

It should have a path like /en/accessibility & /fr/accessibilité because that also helps people find it (and yes /accessibility should redirect to /en/accessibility or /fr/accessibilité (Browsers give the default language of the user so this choice is pretty simple and can be automated. If a user selects a different choice, this can be stored in a cookie. There is of course a link between the en & fr if someone makes the right choice.  In 2022 we shouldn't be asking people if they want to go to the English or French page (or accessibility site).
